tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -95,6 +95,7 @@ def setUpClass(cls):
                 cls.client.create_dataset(dataset, exists_ok=True)
 
         def setUp(self):
+            self.tearDownClass()
```

Comment:
Is this left over from debugging or has the `bigquery_tests_base` been changed and I didn't do a pull when I needed to?

Reply:
I added this because existing tables can interfere with the test. This can help mitigate errors/failures